### PR TITLE
feat: hapi v17

### PR DESCRIPTION
**Pull request by kuryaki**
Do not merge yet

- Adds support for hapi V17, no backward compatibility

### CODE REVIEW

**Review by hamin — APPROVED** [review]

**Review by ryanhallcs — APPROVED** [review]

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  extends: 'hapi',
+  parserOptions: {
+    ecmaVersion: 8
+  }
+}
```
Comment (cilindrox) [inline .eslintrc.js]: missing an EOL char

File: .travis.yml [hunk 2]
```diff
@@ -1,7 +0,0 @@
-language: node_js
```
Comment (cilindrox) [inline .travis.yml]: Why are we ditching CI?

File: lib/index.js [hunk 3]
```diff
@@ -2,52 +2,46 @@
...
-                remoteAddress: request.info.remoteAddress
-            },
-            tags: options.tags
+            Raven.captureException(err, {
+                request: {
+                    method: request.method,
+                    query_string: request.query,
+                    headers: request.headers,
+                    cookies: request.state,
+                    url: baseUrl + request.path
```
Comment (cilindrox) [inline lib/index.js]: let's use `string ${interpolation}` for url here

**Review by cilindrox — CHANGES_REQUESTED** [review]
thanks @kuryaki

File: .travis.yml (hunk 2 above)
Comment (kuryaki) [reply .travis.yml]: hmm thinking that this was not being used by pager, but figured it out this project is OS, returning it back

Comment (cilindrox) [reply .travis.yml]: exactly, thanks @kuryaki !

**Review by cilindrox — APPROVED** [review]

**Review by cilindrox — APPROVED** [review]

**Review by gmauricio — APPROVED** [review]

**Pull request merged**